fiftyone/zoo/models/torch.py

```diff
@@ -0,0 +1,104 @@
+"""
+FiftyOne Zoo models provided by ``torchvision.models``.
+
+| Copyright 2017-2020, Voxel51, Inc.
+| `voxel51.com <https://voxel51.com/>`_
+|
+"""
+import inspect
+
+import eta.core.utils as etau
+
+import fiftyone as fo
+import fiftyone.core.utils as fou
+import fiftyone.utils.torch as fout
+from fiftyone.zoo.models import HasZooModel
+
+fou.ensure_torch()
+import torchvision
+
+
+class TorchvisionImageModelConfig(fout.TorchImageModelConfig, HasZooModel):
+    """Configuration for running a :class:`TorchvisionImageModel`.
+
+    Args:
+        model_name (None): the name of a zoo model containing a state dict to
+            load
+        model_path (None): the path to a state dict on disk to load
+        entrypoint_fcn: a string like ``"torchvision.models.inception_v3"``
+            specifying the entrypoint function that loads the model
+        entrypoint_args (None): a dictionary of arguments for
+            ``entrypoint_fcn``
+        output_processor_cls: a string like
+            ``"fifytone.utils.torch.ClassifierOutputProcessor"`` specifying the
+            :class:`fifytone.utils.torch.OutputProcessor` to use
+        output_processor_args (None): a dictionary of arguments for
+            ``output_processor_cls(class_labels, **kwargs)``
+        labels_string (None): a comma-separated list of the class names for the
+            model
+        labels_path (None): the path to the labels map for the model
+        use_half_precision (None): whether to use half precision
+        image_min_size (None): a minimum ``(width, height)`` to which to resize
+            the input images during preprocessing
+        image_min_dim (None): a minimum image dimension to which to resize the
```

Comment:
This applies to height and width, correct?

Reply:
I clarified a bit:

```
        image_min_size (None): resize the input images during preprocessing, if
            necessary, so that the image dimensions are at least this
            ``(width, height)``
        image_min_dim (None): resize input images during preprocessing, if
            necessary, so that the smaller image dimension is at least this
            value
```